cms/migrations/0006_auto_20160421_1712.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-04-21 17:12
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def set_default_homepage_text(apps, schema_editor):
+    HomePage = apps.get_model("cms", "HomePage")
+    homepage = HomePage.objects.first()
+    homepage.title = """MIT credentials are now attainable by \
+anyone in the world with the discipline, dedication, and \
+intelligence to master course material."""
```

Comment:
You should not use the triple quotes for this: they are used for docstrings.
You can use string literal concatenation for this, so something like:
```
("MIT credentials are now attainable by"
 "anyone in the world with the discipline, dedication, and"
 "intelligence to master course material."
```
Note that there are no commas